src/datagrids/ImportDataTable.tsx

```diff
@@ -153,11 +180,11 @@ function ImportDataTable() {
   };
 
   const [columnDefs] = useState([
-    { field: 'bucket', headerName: 'Bucket', cellRenderer: BucketColumnRenderer, width: 180 },
-    { field: 'status', headerName: 'Status', cellRenderer: StatusColumnRenderer, width: 200 },
-    { field: 'importedBy', headerName: 'Imported By', cellRenderer: ImportedByColumnRenderer },
-    { field: 'importedOn', headerName: 'Imported On', cellRenderer: ImportedOnColumnRenderer },
-    { field: 'importFile', headerName: 'Import File', cellRenderer: ImportFileColumnRenderer, width: 180 },
+    { field: 'bucket', headerName: 'BUCKET', cellRenderer: BucketColumnRenderer, width: 180 },
```

Comment:
I think we can expect that the headers will **always** be uppercased and therefore this should be included in the theme, rather than hard coded